src/components/TreeSpeciesCard.js

```diff
@@ -70,7 +77,7 @@ function TreeSpeciesCard(props) {
           sx={{
             background: (t) => t.palette.background.paperDark,
             color: (t) => t.palette.text.primaryReverse,
-            padding: (t) => [t.spacing(3, 5), t.spacing(6, 8)],
+            padding: (t) => [t.spacing(4.5, 6.5), t.spacing(6, 8)],
```

Comment:
Personally I do not see the need of a padding increase, what is your opinion @jeremydthomas ?

Reply:
I see the issue now, when wordwrapping the text can become a little close to the border

Reply:
I haven't actually looked at it more than a glance to be honest. What I did see seemed better than what I did. I was working on something else tonight, also about to go home here in a bit.